streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -427,8 +428,8 @@ public void singleSourceShouldHaveSingleSubtopology() {
         final TopologyDescription.Source expectedSourceNode = addSource("source", "topic");
 
         expectedDescription.addSubtopology(
-            new InternalTopologyBuilder.Subtopology(0,
-                Collections.singleton(expectedSourceNode)));
+            new SubtopologyDescription(0,
+                                       Collections.singleton(expectedSourceNode)));
```

Comment:
nit: are these indents right?

Reply:
Yeah, looks like my IDE just reformatted this when it did the renaming.